cmd/helm/upgrade.go

```diff
@@ -252,6 +254,7 @@ func newUpgradeCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 
 	f := cmd.Flags()
 	f.BoolVar(&createNamespace, "create-namespace", false, "if --install is set, create the release namespace if not present")
+	f.StringToStringVarP(&namespaceLabels, "namespace-labels", "L", nil, "Labels that would be added to created namespace. Should be divided by comma. If --create-namespace is not set, this is ignored. Default is 'name=<namespace-name>'")
```

Comment:
```suggestion
	f.StringToStringVar(&namespaceLabels, "namespace-labels", nil, "labels to add to the created namespace. Comma separated key=value pairs. Ignored if --create-namespace is not set (default 'name=<namespace-name>'")
```

I would propose no "shorthand" flag here. IMHO, shorthand flags are useful to save typing for commonly used flags. I don't think a `--namespace-labels` flag meets that criteria.

Also some suggestions for the verbiage.